globus_automate_client/cli/flows.py

```diff
@@ -507,14 +507,20 @@ def flow_run(
         case_sensitive=False,
         show_default=True,
     ),
+    label: str = typer.Option(
```

Comment:
Is there a max length that this label can or should be? Transfer caps at 128 but I'm not sure what we're doing on Flows.

Reply:
This crossed my mind on the [API side](https://github.com/globusonline/flows/pull/137) too. 128 sounds like a reasonable limit. In this case I think I'd rather include that on only the API side? since the API already has to also validate the input body. If that makes sense I'll add it to the API PR.

Reply:
[Added length check to that PR](https://github.com/globusonline/flows/pull/137/files#diff-ac504e479ec42f07e22c65fc287e2206cdfb5f63dd20f9eb88764ba9eb66efdeR474-R483)